worlds/jakanddaxter/__init__.py

```diff
@@ -0,0 +1,493 @@
+        "Scout Flies - PB": set(Scouts.locPB_scoutTable.values()),
+        "Scout Flies - LPC": set(Scouts.locLPC_scoutTable.values()),
+        "Scout Flies - BS": set(Scouts.locBS_scoutTable.values()),
+        "Scout Flies - MP": set(Scouts.locMP_scoutTable.values()),
+        "Scout Flies - VC": set(Scouts.locVC_scoutTable.values()),
+        "Scout Flies - SC": set(Scouts.locSC_scoutTable.values()),
+        "Scout Flies - SM": set(Scouts.locSM_scoutTable.values()),
+        "Scout Flies - LT": set(Scouts.locLT_scoutTable.values()),
+        "Scout Flies - GMC": set(Scouts.locGMC_scoutTable.values()),
+        "Specials": set(special_location_table.values()),
+        "Orb Caches": set(cache_location_table.values()),
+        "Precursor Orbs": set(orb_location_table.values()),
+        "Precursor Orbs - GR": set(Orbs.locGR_orbBundleTable.values()),
+        "Precursor Orbs - SV": set(Orbs.locSV_orbBundleTable.values()),
+        "Precursor Orbs - FJ": set(Orbs.locFJ_orbBundleTable.values()),
+        "Precursor Orbs - SB": set(Orbs.locSB_orbBundleTable.values()),
+        "Precursor Orbs - MI": set(Orbs.locMI_orbBundleTable.values()),
+        "Precursor Orbs - FC": set(Orbs.locFC_orbBundleTable.values()),
+        "Precursor Orbs - RV": set(Orbs.locRV_orbBundleTable.values()),
+        "Precursor Orbs - PB": set(Orbs.locPB_orbBundleTable.values()),
+        "Precursor Orbs - LPC": set(Orbs.locLPC_orbBundleTable.values()),
+        "Precursor Orbs - BS": set(Orbs.locBS_orbBundleTable.values()),
+        "Precursor Orbs - MP": set(Orbs.locMP_orbBundleTable.values()),
+        "Precursor Orbs - VC": set(Orbs.locVC_orbBundleTable.values()),
+        "Precursor Orbs - SC": set(Orbs.locSC_orbBundleTable.values()),
+        "Precursor Orbs - SM": set(Orbs.locSM_orbBundleTable.values()),
+        "Precursor Orbs - LT": set(Orbs.locLT_orbBundleTable.values()),
+        "Precursor Orbs - GMC": set(Orbs.locGMC_orbBundleTable.values()),
+        "Trades": {location_table[Cells.to_ap_id(k)] for k in
+                   {11, 12, 31, 32, 33, 96, 97, 98, 99, 13, 14, 34, 35, 100, 101}},
+        "'Free 7 Scout Flies' Power Cells": set(Cells.loc7SF_cellTable.values()),
+    }
+
+    # These functions and variables are Options-driven, keep them as instance variables here so that we don't clog up
+    # the seed generation routines with options checking. So we set these once, and then just use them as needed.
+    can_trade: Callable[[CollectionState, int, int | None], bool]
+    total_orbs: int = 2000
+    orb_bundle_item_name: str = ""
+    orb_bundle_size: int = 0
+    total_trade_orbs: int = 0
+    total_prog_orb_bundles: int = 0
+    total_trap_orb_bundles: int = 0
+    total_filler_orb_bundles: int = 0
+    total_power_cells: int = 101
+    total_prog_cells: int = 0
+    total_trap_cells: int = 0
+    total_filler_cells: int = 0
+    power_cell_thresholds: list[int] = []
+    trap_weights: tuple[list[str], list[int]] = ({}, {})
```

Comment:
you aren't using these as ClassVar so initializing them here isn't a great idea.
```suggestion
    power_cell_thresholds: list[int]
    trap_weights: tuple[list[str], list[int]]
```